kolibri/plugins/coach/assets/src/views/exams-page/preview-exam-modal.vue

```diff
@@ -4,7 +4,7 @@
     <div class="question-selector-container">
       <div class="question-selector">
         <div>
-          {{ $tr('numQuestions', { num: exam.question_count })}}<icon-button :text="$tr('randomize')" @click="$emit('randomize')" :primary="true"/>
```

Comment:
(might also need some modifications to state, but I think I pass most stuff in as a prop).